.github/workflows/publish.yml

```diff
@@ -19,19 +19,20 @@ jobs:
       matrix:
         python:
           - ["3.11", cp311]
-          - ["3.12", cp312]
+          # - ["3.12", cp312]
```

Comment:
Are you keeping the comments?

Reply:
I am just turning them on and off for different PRs to save run time for github runners. I will turn them all on when this is stable